Source/carthage/CopyFrameworks.swift

```diff
@@ -27,11 +27,19 @@ public struct CopyFrameworksCommand: CommandType {
 
 				return combineLatest(SignalProducer(result: source), SignalProducer(result: target), SignalProducer(result: validArchitectures()))
 					.flatMap(.Merge) { (source, target, validArchitectures) -> SignalProducer<(), CarthageError> in
-						let copyFrameworks = copyFramework(source, target: target, validArchitectures: validArchitectures)
-						let copydSYMs = copyDebugSymbolsForFramework(source, validArchitectures: validArchitectures)
-						return combineLatest(copyFrameworks, copydSYMs)
-							.then(.empty)
-					}
+						return shouldIgnoreFramework(source, validArchitectures: validArchitectures)
+							.flatMap(.Concat) { shouldIgnore -> SignalProducer<(), CarthageError> in
+								if shouldIgnore {
+									carthage.println("warning: Ignoring \(frameworkName) because it does not support the current architecture\n")
```

Comment:
I use `warning:` because Xcode shows all printed lines that begin with `warning:` as  a warning:
![screen shot 2016-04-13 at 09 14 00](https://cloud.githubusercontent.com/assets/2033982/14485729/0e742214-0159-11e6-9518-69c78043c64f.png)

Instead of(using Carthage formatting):
![screen shot 2016-04-13 at 09 19 19](https://cloud.githubusercontent.com/assets/2033982/14485737/1cf577de-0159-11e6-8495-2f90f0cf4037.png)